tools/text_processing/text_processing/tool_dependencies.xml

```diff
@@ -1,34 +0,0 @@
-<?xml version="1.0"?>
```

Comment:
Doesn't this mean by default with the "recommended" approach to tool installs - Galaxy will now use the insecure system version of sed?